src/store/reducers/appReducer.js

```diff
@@ -200,7 +243,9 @@ const fetchableTypes = [
   epics.actionTypes.fetchDeviceGroupFilters,
   epics.actionTypes.fetchAzureMapsKey,
   epics.actionTypes.updateLogo,
-  epics.actionTypes.fetchLogo
+  epics.actionTypes.fetchLogo,
+  epics.actions.getSolutionSettings,
```

Comment:
Rename with "fetch" to be consistent with others.

Reply:
fixed